.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+on: ["pull_request"]
+name: Proxy repository
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x, 1.15.x]
+        os: [ubuntu-16.04]
+    runs-on: ${{ matrix.os }}
+    env:
+      GO111MODULE: off
+      TRAVIS: "true"
+      TRAVIS_BRANCH: ${{ github.base_ref }}
+      TRAVIS_PULL_REQUEST_BRANCH: ${{ github.head_ref }}
+      TRAVIS_PULL_REQUEST_SHA : ${{ github.event.pull_request.head.sha }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Setup GOPATH
+      run: |
+        gopath_org=$(go env GOPATH)/src/github.com/kata-containers/
+        mkdir -p ${gopath_org}
+        ln -s ${PWD} ${gopath_org}
+        echo "TRAVIS_BRANCH: ${TRAVIS_BRANCH}"
+        echo "TRAVIS_PULL_REQUEST_BRANCH: ${TRAVIS_PULL_REQUEST_BRANCH}"
+        echo "TRAVIS_PULL_REQUEST_SHA: ${TRAVIS_PULL_REQUEST_SHA}"
+        echo "TRAVIS: ${TRAVIS}"
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Setup travis references
+      run: |
```

Comment:
@amshinde , I modified it this as before we were having the issue of `a step cannot have both the "uses" and "run" "keys"`

Reply:
@amshinde , after the changes, I still see
```
Running checkcommits version 0.0.1 (commit 084d5fb0d421961c1e86768bc8c744e02bf3a291)
Detected TravisCI Environment
ERROR: Failed to run command /usr/bin/git rev-list --no-merges --reverse origin/master..7b871e78c646426d7fdf9c76ddfb9295b4247885: exit status 128 (stdout: , stderr: fatal: ambiguous argument 'origin/master..7b871e78c646426d7fdf9c76ddfb9295b4247885': unknown revision or path not in the working tree.
```